postproengine/wake_meandering.py

```diff
@@ -0,0 +1,392 @@
+# Get the location where this script is being run
+import sys, os
+scriptpath = os.path.dirname(os.path.realpath(__file__))
+basepath   = os.path.dirname(scriptpath)
+# Add any possible locations of amr-wind-frontend here
+amrwindfedirs = ['../',
+                 basepath]
+for x in amrwindfedirs: sys.path.insert(1, x)
+
+from postproengine import registerplugin, mergedicts, registeraction
+import postproamrwindsample_xarray as ppsamplexr
+import postproamrwindsample as ppsample
+import numpy as np
+import pandas as pd
+import os
+import pickle
+import matplotlib.pyplot as plt
+from mpl_toolkits.axes_grid1 import make_axes_locatable
+import importlib.util
+import sys
+
+# Add check for samwich package
+name = 'samwich'
+if name in sys.modules:
+    usesamwhich=True
```

Comment:
This block could cause trouble: even if samwhich is loaded, it might not have the direct imports for `PlanarData` and `track`, might be best to just start with the elif branch.

Reply:
Got it, I removed this if statement